Algorithms/GEMETD.py

```diff
@@ -0,0 +1,56 @@
+from Algorithms.ETDLB import ETDLB
+from Algorithms.BaseTD import BaseTD
+import numpy as np
+
+
+class GEMETD(ETDLB):
+    """
+    An ETD(0) implementation that uses GEM (aka GTD2(0) with x and x_p switched) to estimate emphasis.
+    """
+    def __init__(self, task, **kwargs):
+        super().__init__(task, **kwargs)
+        self.beta = self.task.GAMMA
+        self.gem_alpha = self.get('gem_alpha')  # Step size for GEM weights.
+        self.gem_eta = self.get('gem_eta')  # Regularization parameter for GEM; not needed for a fixed target policy.
+        self.k = np.zeros(self.task.num_features)  # Auxiliary weights for GTD2.
+        self.u = np.zeros(self.task.num_features)  # Main weights for GTD2.
```

Comment:
I think we already have ``self.w`` defined in BaseTD and ETDLB, so we might not need to redefine ``self.u``

Reply:
Yeah, `self.w` is defined in BaseTD and still used for the value function in GEMETD. `self.u` is one of the two new weight vectors used to estimate emphasis. It plays a similar role to `self.w` though, which is why I went with `u`. Would a different variable name be better?

Reply:
Got it. I think u is fine in this case.